docs/webhooks.md

```diff
@@ -0,0 +1,72 @@
+# Webhooks
+
+?> Available `5.2.0`
+
+A variety of webhooks are available to extend the functionality of SQLPad.
+
+To enable a specific webhook, provide a URL for the event you would like to subscribe to. This URL will be sent an HTTP(S) POST with a payload of relevant information when the event occurs.
+
+To ensure a webhook call is valid, `SQLPAD_WEBHOOK_SECRET` may be configured, and validated on every webhook call.
+
+Each webhook will contain the following headers:
+
+- `Content-Type`: `application/json`
+- `SQLPad-Secret`: `SQLPAD_WEBHOOK_SECRET value`
+- `SQLPad-URL`: `http://your-sqlpad-public-url:port/base-url`
+- `SQLPad-Hook-Name`: `hook_name`
+
+The payload sent to the URL varies by event.
+
+## User Created
+
+The user created webhook whenever a user is added via the users API/UI. This hook can be used to automate user invites, sending a message to users inviting them to sign up (if using local user/password authentication) or sign in (if using a single sign on).
```

Comment:
what happens when a user is auto-created (i.e., SAML)

Reply:
I was wondering about that. I suppose it would make sense that it would fire whenever a user is created (even if automatic by whatever auth is used)